src/sidebar/components/excerpt.js

```diff
@@ -12,24 +12,20 @@ import { applyTheme } from '../util/theme';
  * it is expanded or collapsed.
  */
 function InlineControls({ isCollapsed, setCollapsed, linkStyle = {} }) {
-  const toggleTitle = isCollapsed
-    ? 'Show the full excerpt'
-    : 'Show the first few lines only';
   const toggleLabel = isCollapsed ? 'More' : 'Less';
 
   return (
     <div className="excerpt__inline-controls">
       <span className="excerpt__toggle-link">
-        {/* FIXME-A11Y */}
-        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-        <a
-          href="#"
+        <button
```

Comment:
Again, there was a bit of custom bits here  with styling  and I think we can circle back to this in broader context when converting <buttons> to <Buttons>